Champions/Global/SummonerRevive.cs

```diff
@@ -27,10 +28,9 @@ public void OnFinishCasting(Champion owner, Spell spell, AttackableUnit target)
             statMod.MoveSpeed.PercentBonus = 125.0f / 100.0f;
             owner.AddStatModifier(statMod);
             ApiFunctionManager.AddParticleTarget(owner, "Global_SS_Revive.troy", owner);
-            var visualBuff = ApiFunctionManager.AddBuffHudVisual("SummonerReviveSpeedBoost", 12.0f, 1, owner);
+            var visualBuff = ApiFunctionManager.AddBuffHudVisual("SummonerReviveSpeedBoost", 12.0f, 1, BuffType.COMBAT_ENCHANCER, owner, 12.0f);
             ApiFunctionManager.CreateTimer(12.0f, () =>
             {
-                ApiFunctionManager.RemoveBuffHudVisual(visualBuff);
```

Comment:
Is there a downside to not removing the buff hud visual?

Reply:
The final argument in AddBuffHudVisual is a Timer Argument, which will cause the buff HUD visual to be automatically be removed after the set amount of time. This means that the CreateTimer() calls are (in several cases) unneeded extra code.